cmd/server/assets/users/index.html

```diff
@@ -19,69 +19,74 @@
     {{template "flash" .}}
 
     <h1>Users</h1>
-    <p>
-      These are the users of this realm. You can also <a href="/users/new">create a new user</a>.
-    </p>
+    <p>Manage users of <strong>{{.currentRealm.Name}}</strong>.</p>
 
-    {{if $users}}
-    <div class="table-responsive">
-      <table class="table table-bordered table-striped bg-white">
-        <thead>
-          <tr>
-            <th scope="col">Name</th>
-            <th scope="col" width="300">Email</th>
-            <th scope="col" width="75">Options</th>
-            <th scope="col" width="40"></th>
-          </tr>
-        </thead>
-        <tbody>
-          {{range $users}}
-          <tr>
-            <td>
-              <a href="/users/{{.ID}}">{{.Name}}</a>
-            </td>
-            <td>
-              {{.Email}}
-            </td>
-            <td class="text-center">
-              {{if .CanAdminRealm $currentRealm.ID}}
-              <span class="badge badge-pill badge-primary">Admin</span>
+    <div class="card mb-3 shadow-sm" style="min-width:550px;">
```

Comment:
Same note about min-width here. Can we use any of these instead? https://getbootstrap.com/docs/4.0/utilities/sizing/